manifests/gcp_marketplace/deployer/init_action.sh

```diff
@@ -33,15 +32,13 @@ function set_bucket_and_configmap() {
     return 0
   fi
 
-  for i in $(seq 1 ${NUM_RETRIES})
-  do
+  bucket_name="${GCP_PROJECT_ID}-kubeflowpipelines-default"
+  bucket_is_set=true
+  gsutil ls ${bucket_name} || bucket_is_set=false
+  if [ "$bucket_is_set" = false ]; then
     bucket_is_set=true
-    bucket_name="hostedkfp-default-$(cat /dev/urandom | tr -dc 'a-z0-9' | fold -w 10 | head -n 1)"
-    gsutil mb -p ${GCP_PROJECT_ID} "gs://${bucket_name}/" || bucket_is_set=false
-    if [ "$bucket_is_set" = true ]; then
-      break
-    fi
-  done
+    gsutil mb -p ${GCP_PROJECT_ID} "gs://${bucket_name}/" || bucket_is_set=false  
```

Comment:
Are there any concerns removing the retry?

Reply:
the retry is more for try different names

Reply:
I see, SGTM